src/i18n/locales/mr/translation.json

```diff
@@ -768,7 +768,7 @@
     "settings": "सेटिंग्ज",
     "accounts": "खाते",
     "discover": "शोध",
-    "websites": "वेबसाइट्स",
+    "connected_sites": "कनेक्ट केलेल्या साइट्स",
```

Comment:
Where do those translations come from? If those aren't your native languages I'd prefer to keep them empty so native speakers can translate them.

Reply:
I took those from google translate, didn't knew about the translation thing before.

Reply:
yeah sure, doing this